fe49-onl-maryia-dauhan/src/features/hamburger/hamburger.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+type Props = {
+  items: Array<{
+    id: string;
+    title: string;
+  }>;
+  onClick: () => void;
+};
+
+export const Hamburger: React.FC<Props> = ({ items, onClick }) => {
+  const [hamburgerOpen, setHamburgerOpen] = useState(false);
+
+  const toggleHamburger = () => {
+    setHamburgerOpen(!hamburgerOpen);
+  };
+
+  return (
+    <div>
+      <Humburger onClick={toggleHamburger}>
+        <Burger
+          className='burger1'
+          style={{ transform: hamburgerOpen ? 'rotate(45deg)' : 'rotate(0)' }}
+        ></Burger>
+        <Burger
+          className='burger2'
+          style={{ transform: hamburgerOpen ? 'rotate(-45deg)' : 'rotate(0)' }}
+        ></Burger>
+      </Humburger>
+
+      <ListWrapper style={{ display: hamburgerOpen ? 'block' : 'none' }}>
```

Comment:
the logic should be placed inside styled component

Reply:
fixed